cmd/config.go

```diff
@@ -164,13 +163,21 @@ func getHostConfig(reader *bufio.Reader, index int) (*v3.RKEConfigNode, error) {
 	if err != nil {
 		return nil, err
 	}
-	host.SSHKeyPath = sshKeyPath
-
-	sshKey, err := getConfig(reader, fmt.Sprintf("SSH Private Key of host (%s)", address), "")
-	if err != nil {
-		return nil, err
+	if len(sshKeyPath) == 0 {
+		fmt.Printf("[-] You have entered empty SSH key path\n")
```

Comment:
we should perhaps identify our next action on empty ssh key path. Similar the way you've done on line 173